### ISSUE TITLE
False positive when calling a function from an explicit namespace within another function.

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint-scss/stylelint-scss#1112]
False positive when calling a function from an explicit namespace within another function from `declaration-property-value-no-unknown`.
E.g.
```scss
@use 'foo/bar';

.a {
  background-image: linear-gradient(
    to right,
    bar.get('blue') 27%,
    bar.get('red) 73%
  );
}
```
`Cannot parse property value "linear-gradient(
    to right,
    bar.get('blue') 27%,
    bar.get('red) 73%
  );" for property "background-image" (scss/declaration-property-value-no-unknown)`
